client/sites-dashboard/components/sites-ellipsis-menu.tsx

```diff
@@ -204,8 +189,6 @@ const CopySiteItem = ( { recordTracks, site }: SitesMenuItemProps ) => {
 		<MenuItemLink
 			href={ copySiteHref }
 			onClick={ () => {
-				clearSignupDestinationCookie();
-				setPlanCartItem( { product_slug: plan.product_slug } );
 				recordTracks( 'calypso_sites_dashboard_site_action_copy_site_click' );
```

Comment:
I actually like how this was before. It feels a little strange pulling code from the `/client/landing/stepper` area of the code into the site's dashboard. And as far as I can tell there's nowhere else using the `startSiteCopy` that means it needs to be shared.

Reply:
We did it like this to avoid writing the same code twice. This code is used exactly as is to the `Settings > General > Site Tools`. And this is the reason we refactor it to the hook in the first place.